algorithms/unit_tests/TestBinSortA.hpp

```diff
@@ -19,8 +19,14 @@
 
 #include <gtest/gtest.h>
 #include <Kokkos_Core.hpp>
+#include <Kokkos_Macros.hpp>
+#ifdef KOKKOS_ENABLE_EXPERIMENTAL_CXX20_MODULES
+import kokkos.random;
+import kokkos.sort;
+#else
```

Comment:
Approving but noting that this is the pattern that I was explicitly I don't want to let proliferate thorough the code base

Reply:
I think we agreed in https://github.com/kokkos/kokkos/pull/8132 that we rather do that than generating files on the fly for testing.